### ISSUE TITLE
[Accessibility][TextField] - Disabled TextField does not work with screen reader

### ISSUE THREAD

**Issue by miczhongMSFT** [OfficeDev/office-ui-fabric-react#7639]
Screen reader cannot access the value inside of TextField if the TextField is disabled. The expected behavior should be similar to that of a ComboBox. When a ComboBox is disabled, it still allows screen reader to tab into it's <input> field for accessibility.

**Comment by miczhongMSFT** [comment]
In this particular scenario, we have a default value assigned to the TextField, and we do not want users to change this value. We tried using readOnly, however that does not provide a grey-style visual cue to signal users that the text box is not editable.

**Comment by mikewheaton** [comment]
This is by design: https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Attribute/disabled

The Fabric React TextField component matches the behavior of a standard HTML input, which is to not be keyboard focused when disabled. Here's a CodePen demonstrating that: https://codepen.io/mikewheaton/pen/GPzrLz?editors=0010
